app/assets/stylesheets/scaffold.css

```diff
@@ -0,0 +1,84 @@
+body {
```

Comment:
I think you can use a sprockets directive, they look like this.
```js
//= gulp/filename
```

There's a bunch in `app/assets/javascripts/application.js`. Idk if there is a more elegant solution than replacing this file with just a sprockets directive.